Makefile

```diff
@@ -37,7 +37,7 @@ all::
 	$(call MSBUILD_BINLOG,all,$(_SLN_BUILD)) $(MSBUILD_FLAGS) $(SOLUTION)
 
 all-tests::
-	MSBUILD="$(MSBUILD)" $(call MSBUILD_BINLOG,all-tests,tools/scripts/xabuild) $(MSBUILD_FLAGS) Xamarin.Android-Tests.sln
+	MSBUILD="$(MSBUILD)" $(call MSBUILD_BINLOG,all-tests,tools/scripts/xabuild) $(MSBUILD_FLAGS) Xamarin.Android-Tests.sln /restore
```

Comment:
I feel like this doesn't belong here, *especially* in a possible future world order (current world order?) of using `@(PackageReference)` and .NET Standard 2 (https://github.com/xamarin/java.interop/pull/393).  It feels like this should be part of `make prepare`:

```diff
diff --git a/Makefile b/Makefile
index 0d9b6646..d04a2a51 100644
--- a/Makefile
+++ b/Makefile
@@ -102,6 +102,8 @@ prepare-external:
        git submodule update --init --recursive
        nuget restore $(SOLUTION)
        nuget restore Xamarin.Android-Tests.sln
+       $(MSBUILD) /restore $(SOLUTION)
+       $(MSBUILD) /restore Xamarin.Android-Tests.sln
        (cd external/xamarin-android-tools && make prepare CONFIGURATION=$(CONFIGURATION))
        (cd $(call GetPath,JavaInterop) && make prepare CONFIGURATION=$(CONFIGURATION) JI_MAX_JDK=8)
        (cd $(call GetPath,JavaInterop) && make bin/Build$(CONFIGURATION)/JdkInfo.props CONFIGURATION=$(CONFIGURATION) JI_MAX_JDK=8)
```